Controller.php

```diff
@@ -719,4 +728,20 @@ public static function getComponentExtensions($isNoDataPage = false): array
         ]);
         return $componentExtensions;
     }
+
+    private function getStreamIdsFromRequest()
+    {
+        $streamIds = Common::getRequestVar('streamIds', [], $type = 'array');
```

Comment:
I think that going forward, we're supposed to be using the `Piwik\Request` class to get query parameters instead of `Common::getRequestVar`.

Reply:
Changed to `$request->getArrayParameter`